jerry-core/ecma/builtin-objects/ecma-builtin-array-prototype.cpp

```diff
@@ -1117,6 +1120,9 @@ ecma_builtin_array_prototype_object_last_index_of (ecma_value_t this_arg, /**< t
      * for an underflow instead. This is safe, because from_idx will always start in [0, len - 1],
      * and len is in [0, UINT_MAX], so from_idx >= len means we've had an underflow, and should stop.
      */
+    /* FIXME: Iterating over the whole lenght is unnecessary in case of sparse arrays,
+     *        and could take a long time if len is large. We should only iterate
+     *        over defined array indexes. */
```

Comment:
Could you, please, also move the changes, adding `FIXME` to separate commit in the pull request?